### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by SkySkimmer**
I was quite confused when make started complaining about missing git submodules.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,10 +18,7 @@ ifeq "$(ELPIDIR)" ""
 ELPIDIR=$(shell which elpi >/dev/null 2>&1 && elpi -where)
 endif
 ifeq "$(ELPIDIR)" ""
-ELPIDIR=elpi/findlib/elpi
-$(info Using elpi from the git submodule ./elpi/, override via ELPIDIR or PATH)
-else
-$(info Using elpi found in $(ELPIDIR), from ELPIDIR or PATH)
+$(error Elpi not found, make sure it is installed)
```
Comment (gares) [inline Makefile]: `make sure elpi is in your path or set ELPIDIR`

**Comment by gares** [comment]
thanks!

**Comment by SkySkimmer** [comment]
The submodule was removed in 6c65db3, this is followup.

**Comment by gares** [comment]
ok, I'll merge then and fix the message myself

File: Makefile (hunk 1 above)
Comment (SkySkimmer) [reply Makefile]: Changed.

Comment (SkySkimmer) [reply Makefile]: 1 minute too late :D

**Pull request merged**